### ISSUE TITLE
HLR Production Support

### ISSUE THREAD

**Issue by michelpmcdonald** [department-of-veterans-affairs/va.gov-team#42664]
Backend tracking ticket for HLR prod issues.

Around 50 HLRs were rejected for submission from Lighthouse.

After pulling the reject claims, 2 ongoing issues were discovered and corrected, they were fixed in the front end 

International addresses in HLRs\NODs now require a US zipcode(Field name "zipCode5") of "00000"

Date Control on HLR issues page was allowing users to enter 'decisionDate'(s) with missing months or days for user supplied issues

PR for Front End work to correct:
https://github.com/department-of-veterans-affairs/vets-website/pull/21316

**Comment by cohnjesse** [comment]
@michelpmcdonald Ok, so what we need to do is get the email addresses for each of the 50 Veterans with rejected HLRs.

@Mottie Can you and I get together and get some screenshots and steps for what these Veterans will need to do to re-submit their HLRs with good data?

**Comment by Mottie** [comment]
[HLR decision date issue document](https://docs.google.com/document/d/1KtG2w6_PjZyArQdyR4KaBs9Le6kPpzE77gr3y1fa78w/edit?usp=sharing) is ready for review

**Comment by sortizsh** [comment]
Need to format Robin's document in the Claim Center template, and notify claim center of users that were affected by issue

**Comment by michelpmcdonald** [comment]
Follow Up, the root cause of both issues(Missing zipcode5 for international addresses, web date control allowing users to create invalid dates) has been corrected on the web side.

This ticket deals with correcting the HLRs that hit the issue.

I was able to correct and resubmit the claims with the missing zip.

For the users who hit the malformed date, there were about ~5 users that hit the issue, however, the table where the errors were stored, personal_information_log has been wiped of old records, so I'm not able to notify those users because I can't find them.

Closing this ticket
